css-overflow-4/Overview.bs

```diff
@@ -1140,6 +1163,44 @@ Fragmentation of Overflow: the 'continue' property</h3>
 	as the box would be a [=flex container=] rather than a [=block container=],
 	and thus the 'continue' property would not apply.
 
+<h4 id=line-clamp-containers>Line-clamp containers</h4>
+
+	A <dfn>line-clamp container</dfn> is an [=independent formatting context|independent=] [=block formatting context=] that
+	additionally follows the rules in this section. All line-clamp containers contain a
+	<dfn>clamp point</dfn>, which is one of the following positions inside it:
+
+	- A point immediately after a [=line box=] in the line-clamp container's [=block formatting
+	  context=].
+	- The end of the line-clamp container, if there are any boxes after the last [=inline formatting
+	  context=] in the line-clamp container's [=block formatting context=].
```

Comment:
As discussed in an earlier comments, I am not sure I understand what the difference/benefit is of placing a clamp point at the end, instead of saying there's no clamp point. We can define that a clamp point at the end has no effect (which I what I believe the definition bellow does), but would it not be simpler to just say that there isn't one?